io.github.tntwise.REAL-Video-Enhancer.desktop

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env xdg-open
+[Desktop Entry]
+Exec=REAL-Video-Enhancer
+Icon=io.github.tntwise.REAL-Video-Enhancer
+Version=1.0.4
```

Comment:
this is NOT the application version. This value is invalid.

Reply:
removed, thank you